<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Test Suite" duration-ms="58" started-at="2019-09-19T18:37:26Z" finished-at="2019-09-19T18:37:26Z">
    <groups>
    </groups>
    <test name="Tests" duration-ms="58" started-at="2019-09-19T18:37:26Z" finished-at="2019-09-19T18:37:26Z">
      <class name="tests.TestsWithoutCucumber">
        <test-method status="PASS" signature="testValuesAmountInFile()[pri:1, instance:tests.TestsWithoutCucumber@4c203ea1]" name="testValuesAmountInFile" duration-ms="31" started-at="2019-09-19T18:37:26Z" finished-at="2019-09-19T18:37:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValuesAmountInFile -->
        <test-method status="PASS" signature="testValueGreaterThanZero()[pri:2, instance:tests.TestsWithoutCucumber@4c203ea1]" name="testValueGreaterThanZero" duration-ms="1" started-at="2019-09-19T18:37:26Z" finished-at="2019-09-19T18:37:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValueGreaterThanZero -->
        <test-method status="FAIL" signature="testTotalBalance()[pri:3, instance:tests.TestsWithoutCucumber@4c203ea1]" name="testTotalBalance" duration-ms="1" started-at="2019-09-19T18:37:26Z" finished-at="2019-09-19T18:37:26Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Summary of all values ($997000.0) doesn't match with expected Total Balance. expected [1000000.0] but found [997000.0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Summary of all values ($997000.0) doesn't match with expected Total Balance. expected [1000000.0] but found [997000.0]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at tests.TestsWithoutCucumber.testTotalBalance(Tests.java:46)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testTotalBalance -->
        <test-method status="PASS" signature="testValuesFormattedAsCurrencies()[pri:4, instance:tests.TestsWithoutCucumber@4c203ea1]" name="testValuesFormattedAsCurrencies" duration-ms="0" started-at="2019-09-19T18:37:26Z" finished-at="2019-09-19T18:37:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValuesFormattedAsCurrencies -->
      </class> <!-- tests.TestsWithoutCucumber -->
    </test> <!-- Tests -->
  </suite> <!-- Test Suite -->
</testng-results>
